tests/Fixer/Phpdoc/NoSuperfluousPhpdocTagsFixerTest.php

```diff
@@ -212,6 +212,42 @@ function foo(Baz $bar) {}',
  */
 function foo(Bar $bar) {}',
             ],
+            'with_param_superfluous_but_not_return' => [
+                '<?php
+class Foo {
+    /**
+     *
+     * @return Baz
+     */
+    public function doFoo(Bar $bar) {}
+}',
+                '<?php
+class Foo {
+    /**
+     * @param Bar $bar
+     *
+     * @return Baz
+     */
+    public function doFoo(Bar $bar) {}
+}',
+            ],
+            'with_not_all_params_superfluous' => [
+                '<?php
+class Foo {
+    /**
+     * @param Bax|Baz $baxz
```

Comment:
so...
usually, in repos I saw, ppl like to have all `@param`+`@return` if at least one of param/return can't be done on language level, and remove whole docblock when all of them are typehinted.

would it be big thing to add such option ?

Reply:
I don't think so, but I would postpone that to a separate PR because that would still require some code and review time and I do want this PR to be merged before next minor release :)

Reply:
that's the thing. I would like to have such option already in 2.12